onnxruntime/test/providers/internal_testing/internal_testing_ep_static_kernels.cc

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
What does "internal_testing" mean?

Reply:
It's an EP that is used for testing EP infrastructure that works on all platforms. e.g. it's used for things like load of ORT model with pseudo-compilation of nodes, the NCHW/NHWC layout transform, etc.. This can't be done easily with the 'real' EPs like NNAPI or CoreML given those only run on Android/iOS.